T/t8code/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "t8code"
+version = v"1.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/DLR-AMR/t8code/releases/download/v$(version)/t8code_v$(version).tar.gz", "01ceae6dd0ed596cf1abb5bc78caa77b5da94ca60d080aae3ccf0ba3c6464deb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd t8code/
+./configure --enable-mpi CFLAGS=-O3 CXXFLAGS=-O3 CC=mpicc CXX=mpicxx --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j
```

Comment:
```suggestion
make -j${nproc}
```
Based on https://docs.binarybuilder.org/stable/build_tips/#Autoconfigure-builds